docs/pages/index.tsx

```diff
@@ -192,6 +190,26 @@ const Home = () => {
               Read
             </HomeButton>
           </GridCell>
+          <GridCell
+            xl={6}
+            lg={6}
+            style={{ backgroundColor: palette.yellow['000'], borderColor: palette.yellow['300'] }}>
+            <OfficeHoursImage />
+            <H3 style={{ color: palette.yellow['900'], marginBottom: spacing[1.5] }}>
+              Join us for Office Hours
+            </H3>
+            <P style={{ color: palette.yellow['800'], ...typography.fontSizes[14] }}>
+              Get answers to your questions and
+              <br />
+              get advice from the Expo team.
+            </P>
+            <HomeButton
+              style={{ backgroundColor: palette.yellow['900'], color: palette.yellow['000'] }}
+              href="https://us02web.zoom.us/meeting/register/tZcvceivqj0oHdGVOjEeKY0dRxCRPb0HzaAK"
+              target="_new">
```

Comment:
```suggestion
              target="_blank">
```

Reply:
We use "_new" for other external links on the Home page, so I assume that we should update them too, right?